lib/WWW/Mechanize.pm

```diff
@@ -223,6 +238,30 @@ history.
 
 =back
 
+In addition, WWW::Mechanize also allows you to globally enable
+strict and verbose mode for form handling, which is done with L<HTML::Form>.
+
+=over 4
+
+=item * C<< strict_forms => [0|1] >>
+
+Globally sets the HTML::Form strict flag which causes form submission to
+croak if any of the passed fields don't exist on the page, and/or a value
```

Comment:
I think neither is fully correct. This is not about searching for forms as far as I understand. It's for once it's found a form, that form will be strict, so it will not silently ignore if you try to pass in extra fields. It doesn't matter how it found that form. If you use a method that will find a form based on fields, then it simply won't find it if you ask for superfluous (or maybe misspelled) fields. That behaviour hasn't changed, it was already there.

I believe saying _any of the passed fields don't exist on the **form**_ makes more sense.